### PR TITLE
Project filters

### PR DESCRIPTION
**Pull request by oorion**

### CODE REVIEW

File: app/models/project.rb [hunk 1]
```diff
@@ -30,6 +30,23 @@ def formatted_dollar_amount
     number_to_currency(price / 100.00)
   end
 
+  def downcase_categories
+    categories.map do |category|
+      category.name.downcase
+    end
+  end
+
+  def as_json(s)
```
Comment (Jwan622) [inline app/models/project.rb]: You need that s to override the as_json method hound. Dumb dog.

File: test/integration/project_filter_test.rb [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+   # page.driver.block_unknown_urls
+    #page.driver.allow_url("maxcdn.bootstrapcdn.com")
+
+    project1 = create(:project)
+    project2 = create(:project)
+    project1_category = project1.categories.first.name
+    project2_category = project2.categories.first.name
+    visit root_path
+
+    binding.pry
```
Comment (Jwan622) [inline test/integration/project_filter_test.rb]: got it.

File: Gemfile [hunk 3]
```diff
@@ -65,4 +66,7 @@ group :development, :test do
   gem 'capybara',               '~> 2.4.4'
   gem 'mocha',                  '~> 1.1.0'
   gem 'launchy',                '~> 2.4.3'
+  gem 'capybara_minitest_spec'
+  # gem 'selenium-webdriver', '2.44.0'
```
Comment (EmilyMB) [inline Gemfile]: Maybe clean up? Remove or use?

File: Gemfile [hunk 4]
```diff
@@ -43,6 +43,7 @@ gem "less-rails",               '~> 2.6.0'
 gem "rails_12factor",           '~> 0.0.3'
 gem 'paperclip',                '~> 4.2.1'
 gem 'faker',                    '~> 1.4.2'
+gem 'aws-sdk'
```
Comment (EmilyMB) [inline Gemfile]: Just to be safe...add version!

File: app/controllers/projects_controller.rb [hunk 5]
```diff
@@ -1,10 +1,11 @@
 class ProjectsController < ApplicationController
   def index
-    @projects = all_projects.select do |project|
```
Comment (EmilyMB) [inline app/controllers/projects_controller.rb]: Don't delete this please!

**Pull request closed without merging**